src/module-elasticsuite-catalog/etc/import.xml

```diff
@@ -0,0 +1,26 @@
+<!--
+/**
+ * Smile_ElasticsuiteCatalog import configuration
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade Smile ElasticSuite to newer
+ * versions in the future.
+ *
+ * @category  Smile
+ * @package   Smile\ElasticsuiteCatalog
+ * @author    Vadym Honcharuk <vahonc@smile.fr>
+ * @copyright 2023 Smile
+ * @license   Open Software License ("OSL") v. 3.0
+ */
+-->
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:noNamespaceSchemaLocation="urn:magento:module:Magento_ImportExport:etc/import.xsd">
+    <entity name="catalog_product"
```

Comment:
entity_name should not be "catalog_product", it should be something new. Maybe "elasticsuite_product_attribute".